PATCH-P1-FIX-UPSTREAM: Change test result default charset to utf8 
BUGS: bnc#937787, MDEV-8486

Default value of the 'DEFAULT_CHARSET' option is 'latin1', but we use
'utf8'. Therefore the test result has to be adjusted accordingly.

Maintainer: Kristyna Streitova <kstreitova@suse.com>

--- mariadb-10.0.20/mysql-test/r/bootstrap.result	
+++ mariadb-10.0.20/mysql-test/r/bootstrap.result	
@@ -20,7 +20,7 @@ show create table t1;
 Table	Create Table
 t1	CREATE TABLE `t1` (
   `a` int(11) DEFAULT NULL
-) ENGINE=MyISAM DEFAULT CHARSET=latin1
+) ENGINE=MyISAM DEFAULT CHARSET=utf8
 drop table t1;
 select * from mysql.plugin;
 name	dl
